src/app/rpmostree-builtin-cleanup.cxx

```diff
@@ -81,7 +81,15 @@ rpmostree_builtin_cleanup (int             argc,
   if (opt_rollback)
     g_ptr_array_add (cleanup_types, (char*)"rollback-deploy");
   if (opt_repomd)
-    g_ptr_array_add (cleanup_types, (char*)"repomd");
+    {
+      auto is_bare_split_xattrs = CXX_TRY_VAL(is_bare_split_xattrs(), error);
+      if (rpmostreecxx::running_in_container() && is_bare_split_xattrs) 
```

Comment:
At this point we should probably add an API that unifies these like `rpmostreecxx::is_ostree_container()`.  (But it's good to have them separate too)

Reply:
https://github.com/ostreedev/ostree-rs-ext/pull/197